src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,36 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	public static final int MAX = 12;
+	public static final int FINAL = -1;
+	private final int[] numbers = new int[MAX];
+
+	public int total = FINAL;
```

Comment:
This constant should be different (separate) than the one in peekaboo and countOut methods.